sensor_msgs/Image input_image
---
sensor_msgs/Image output_image
std_msgs/String objects_json